cmd/oras/push.go

```diff
@@ -145,6 +158,18 @@ func runPush(opts pushOptions) error {
 	}
 
 	fmt.Println("Pushed", opts.targetRef)
+
+	contentBytes, err := content.FetchAll(ctx, store, desc)
+	if err != nil {
+		return err
+	}
```

Comment:
We don't need to fetch if `len(opts.extraRefs)` is 0.

Reply:
good point!